docs/en/01_configuration.md

```diff
@@ -98,9 +105,9 @@ the second part of the path should be the service name.
 
 The values for `service_name_1`, `service_name_2` and `service_name_3` should fit in these sentences:
 
-* `service_name_1`: "To access the [online service], you need a RealMe login."
-* `service_name_2`: "To log in to [this service] you need a RealMe login."
-* `service_name_3`: "[This service] uses RealMe login to secure and protect your personal information."
+- `service_name_1`: "To access the [online service], you need a RealMe login."
+- `service_name_2`: "To log in to [this service] you need a RealMe login."
+- `service_name_3`: "[This service] uses RealMe login to secure and protect your personal information."
 
 **Note:** None of these are required for the assert form, as they are not used (it only uses organisation name, which is
 taken from the `metadata_organisation_name` config value instead.
```

Comment:
```suggestion
> [!NOTE] None of these are required for the assert form, as they are not used (it only uses organisation name, which is
> taken from the `metadata_organisation_name` config value instead.
```
I won't put a comment on every one of these - please update them all.

Reply:
Updated